pgstore.go

```diff
@@ -13,6 +13,7 @@ import (
 	_ "github.com/lib/pq"
 )
 
+// PGStore main type
```

Comment:
Should say "PGStore represents the currently configured session store" or something along those lines.